AstronomyUITests/AstronomyUITests.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  AstronomyUITests.swift
+//  AstronomyUITests
+//
+//  Created by Cora Jacobson on 9/24/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import XCTest
+
+class AstronomyUITests: XCTestCase {
+    
+    let app = XCUIApplication()
+    
+    private var nextSolButton: XCUIElement {
+        return app.buttons["PhotosCollectionViewController.NextSolButton"]
+    }
+    
+    private var previousSolButton: XCUIElement {
+        return app.buttons["PhotosCollectionViewController.PreviousSolButton"]
+    }
+    
+    private var savePhotoButton: XCUIElement {
+        return app.buttons["PhotoDetailViewController.SaveButton"]
+    }
+    
+    private var firstChild: XCUIElement {
+        return app.collectionViews.children(matching: .cell).element(boundBy: 0)
+    }
+    
+    override func setUp() {
+        super.setUp()
+        app.launchArguments = ["UITesting"]
+        app.launch()
+        continueAfterFailure = false
+    }
+
+    func testNextSol() {
```

Comment:
One thing I would say is try to name your tests to indicate the expected outcome. In this case `testNextSolButtonNavigatesToSol16()` or `testTappingNextSolButtonLoadsSol16` or something like that would be appropriate. I know it makes the function name feel super long, but you are never actually calling this function yourself. 

Some people will even use underscores mixed with camel case (you might find it gross, or maybe you'll like it, idk).

> “One trick you would do well to apply is to look for a verb outside of “test” – something saying the action that is 
> happening in the test. For example, testDeathStarFiredLaser() or testMeaningOfLifeShouldBe42() – both make it clear what we expect to happen, and “should” in particular makes a particularly clear assertion.
> 
> As your skill with testing increases, you might find it useful to adopt Roy Osherove’s naming convention for tests: [UnitOfWork_StateUnderTest_ExpectedBehavior]. If you follow that precisely it would create test method names like this: test_Hater_AfterHavingAGoodDay_ShouldNotBeHating().
> 
> Note: Mixing PascalCase and snake_case might hurt your head at first, but at least it makes clear the UnitOfWork – StateUnderTest – ExpectedBehavior separation at a glance. You might also see camelCase being used, which would give test_Hater_afterHavingAGoodDay_shouldNotBeHating()”

Excerpt From: Paul Hudson. “Testing Swift.” Apple Books.